controllers/admin/AdminProductsController.php

```diff
@@ -1557,6 +1557,8 @@ public function ajaxProcessDeleteProductImage($id_image = null)
             $res &= @unlink(_PS_TMP_IMG_DIR_ . 'product_mini_' . $image->id_product . '_' . $this->context->shop->id . '.jpg');
         }
 
+        Hook::exec('actionProductImageDeleted', ['id_image' => $image->id]);
+
```

Comment:
We are not sure that image has been deleted. Place it just before 
```php
$this->jsonConfirmation($this->_conf[7]);

Reply:
Approved with @PululuK